README.md

```diff
@@ -31,7 +31,7 @@ On Windows: Follow the fist two steps [here](https://help.github.com/en/github/c
 
 Run the following command in your shell to make the Wolfram Language engine available to Jupyter:
 
-	.\configure-jupyter.wls add
+	./configure-jupyter.wls add
```

Comment:
This is the Windows/PowerShell section, so isn't `\` the right path delimiter?

Reply:
Oh, you're absolutely right! I saw the macOS at the top near the section heading and didn't notice the later comment saying these were the Windows instructions. Closing without merging.